CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 5.5.1-SNAPSHOT (future release)
+## 6.0.0-SNAPSHOT (future release)
+
+** Highlights **
+- BREAKING CHANGE: add started timestamp to workflow instance table
```

Comment:
Add something about why these changes were added. e.g Faster queries when there is a lot of actions/state variable changes

Reply:
Updated